pallets/weights/src/pallet_asset.rs

```diff
@@ -1,24 +1,130 @@
-//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0-rc6
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
 
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
+use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
 
 pub struct WeightInfo;
 impl pallet_asset::WeightInfo for WeightInfo {
-	// WARNING! Some components were not used: ["t"]
-	fn register_ticker() -> Weight {
-		(50848000 as Weight)
-			.saturating_add(DbWeight::get().reads(10 as Weight))
-			.saturating_add(DbWeight::get().writes(3 as Weight))
-	}
-	// WARNING! Some components were not used: ["f"]
-	fn create_asset(n: u32, i: u32, ) -> Weight {
-		(96128000 as Weight)
-			.saturating_add((35000 as Weight).saturating_mul(n as Weight))
-			.saturating_add((74000 as Weight).saturating_mul(i as Weight))
-			.saturating_add(DbWeight::get().reads(12 as Weight))
-			.saturating_add(DbWeight::get().writes(11 as Weight))
-	}
+    // WARNING! Some components were not used: ["t"]
+    fn register_ticker() -> Weight {
```

Comment:
I think need to bump up the `t` value to see the effect

Reply:
I've removed `t` directly because TICKER_LEN is fixed to 12. We don't need it.